stable/ark/values.yaml

```diff
@@ -0,0 +1,53 @@
+# Default values for ark
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+configuration:
+  # cloudprovider and bucket are always needed
+  cloudprovider: ChangeMe
+  bucket: ChangeMe
+  # region is aws only
+  region: ChangeMe
+  # apitimeout is azure only
+  apitimeout: ChangeMe
+  # credentials not needed if kube2iam is enabled
+  credentials: ChangeMe
+  # following are default values
+  backupSyncPeriod: 60m
+  gcSyncPeriod: 60m
+  scheduleSyncPeriod: 1m
+  restoreOnlyMode: false
+
+image:
+  repository: gcr.io/heptio-images/ark
+  tag: v0.8.1
+  pullPolicy: IfNotPresent
+
+# this must be created manually before helm install, see README
+secret:
```

Comment:
Why not have an option to create the secret and alternatively configure an `existingSecret`?

Reply:
I would rather let the user create their own secrets, it looks to me too much to create this secret from helm